src/model.js

```diff
@@ -2950,6 +2950,7 @@ Specify a different name for either index to resolve this issue.`);
       force: false,
       cascade: false,
       restartIdentity: false,
+      returing: false,
```

Comment:
Typo

Reply:
My bad. Will fix it,  along with changes for review comments.